CryptoPrices/Sources/MyCoin/Sources/MyCoin/CurrentPriceSectionView/CurrentPriceSection.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  CurrentPriceSection.swift
+//  MyCoin
+//
+//  Created by Minh Pham on 22/12/2022.
+//
+
+import Styleguide
+import SwiftUI
+
+struct CurrentPriceSection: View {
+
+    var body: some View {
+        VStack(spacing: 8.0) {
+            // TODO: - Remove dummy
+            Images.icEth.swiftUIImage
+                .resizable()
+                .aspectRatio(contentMode: .fit)
+                .frame(width: 60.0, height: 60.0)
+                .clipShape(Circle())
+
+            VStack(spacing: 4.0) {
+                // TODO: Remove hard-coded data
+                Text(7_273_291, format: .dollarCurrency)
+                    .foregroundColor(Colors.textBold.swiftUIColor)
+                    .font(Fonts.Inter.semiBold.textStyle(.title))
+                    .frame(height: 34.0)
+
+                Button(action: {}, label: {
+                    HStack {
+                        Images.icArrowUpGreen.swiftUIImage
+                        // TODO: Remove hard-coded data
+                        Text(2.41, format: .percentage)
+                            .font(Fonts.Inter.medium.textStyle(.callout))
+                        
```

Comment:
I think we can remove this empty line. 🙏

Reply:
Oh, it is intentional to place a line break here just for better readability, so I would prefer to keep it 😁